### PR TITLE
Fix VM-Install-Shortcut RunAsAdmin argument

### PR DESCRIPTION
**Pull request by emtuls**
This fixes https://github.com/mandiant/VM-Packages/issues/839

### CODE REVIEW

**Review by mr-tz — APPROVED** [review]
good catch

**Review by Ana06 — CHANGES_REQUESTED** [review]
Good catch @emtuls!

I think we should define an arguments variable and add the `RunAsAdmin` if it is `true` instead of duplicating the calls. 🤔

**Comment by emtuls** [comment]
> Good catch @emtuls!
> 
> I think we should define an arguments variable and add the `RunAsAdmin` if it is `true` instead of duplicating the calls. 🤔

Sounds good! Made the change. 👍

**Review by emtuls — COMMENTED** [review]
Adjusted as suggested!

File: packages/common.vm/tools/vm.common/vm.common.psm1 [hunk 1]
```diff
@@ -289,9 +289,30 @@ function VM-Install-Shortcut{
...
-        Install-ChocolateyShortcut -ShortcutFilePath $shortcut -TargetPath $executablePath -RunAsAdmin $runAsAdmin
+        $flagsAndArgs = @{
+            ShortcutFilePath = $shortcut
+            TargetPath       = $executablePath
+        }
+        if ($runAsAdmin) {
+            $flagsAndArgs.RunAsAdmin = $true
+        }
+
+        Install-ChocolateyShortcut $flagsAndArgs
```
[the remark this answers is not in the document]
Comment (emtuls) [reply packages/common.vm/tools/vm.common/vm.common.psm1:315]: Whoops! I missed testing that case it looks like. I'll fix that!

File: packages/common.vm/tools/vm.common/vm.common.psm1 [hunk 2]
```diff
@@ -289,9 +289,30 @@ function VM-Install-Shortcut{
         $executableCmd  = Join-Path ${Env:WinDir} "system32\cmd.exe" -Resolve
         # Change to executable dir, print command to execute, and execute command
         $executableArgs = "/K `"cd `"$executableDir`" && echo $executableDir^> $executablePath $arguments && `"$executablePath`" $arguments`""
-        Install-ChocolateyShortcut -ShortcutFilePath $shortcut -TargetPath $executableCmd -Arguments $executableArgs -WorkingDirectory $executableDir -IconLocation $executablePath -RunAsAdmin $runAsAdmin
+
+        $packageArgs = @{
```
Comment (Ana06) [inline packages/common.vm/tools/vm.common/vm.common.psm1:293]:
```suggestion
        $shortcutArgs = @{
```

**Review by Ana06 — CHANGES_REQUESTED** [review]

**Review by Ana06 — APPROVED** [review]

**Pull request merged**